src/Core/src/Core/ILayout.cs

```diff
@@ -6,7 +6,7 @@ namespace Microsoft.Maui
 	/// Provides the base properties and methods for all Layout elements.
 	/// Use Layout elements to position and size child elements in .NET MAUI applications.
 	/// </summary>
-	public interface ILayout : IView, IContainer
+	public interface ILayout : IView, IContainer, ISafeAreaView
```

Comment:
What is the benefit of having this on `ILayout` as opposed to `IView`? 

I am thinking that for most people, a `IScrollView` might be the root? Also, for a game/drawn app, the root may be a canvas/graphics view?